Scalar.Common/Git/GitProcess.cs

```diff
@@ -148,10 +148,35 @@ public static bool TryGetVersion(string gitBinPath, out GitVersion gitVersion, o
                 return false;
             }
 
+            gitVersion.Features.UnionWith(GetAdvertisedFeatures(gitBinPath));
+
             error = null;
             return true;
         }
 
+        public static IEnumerable<string> GetAdvertisedFeatures(string gitBinPath)
+        {
+            GitProcess gitProcess = new GitProcess(gitBinPath, null);
+            Result result = gitProcess.InvokeGitOutsideEnlistment("version --build-options");
+            string version = result.Output;
+
+            if (result.ExitCodeIsFailure)
+            {
+                yield break;
+            }
+
+            foreach (string line in result.Output.Split('\n', StringSplitOptions.RemoveEmptyEntries))
+            {
+                string prefix = "feature: ";
+                string trimmed = line.Trim();
```

Comment:
`const` and lift to outer scope (I mean, hopefully `csc` is smart enough to do this already...)

```suggestion
            const string prefix = "feature: ";
            foreach (string line in result.Output.Split('\n', StringSplitOptions.RemoveEmptyEntries))
            {
                string trimmed = line.Trim();
```